packages/@react-stately/combobox/src/useComboBoxState.ts

```diff
@@ -19,18 +19,29 @@ import {useEffect, useMemo, useRef, useState} from 'react';
 import {useMenuTriggerState} from '@react-stately/menu';
 
 export interface ComboBoxState<T> extends SelectState<T> {
+  /** The current value of the ComboBox input. */
   inputValue: string,
+  /** Sets the value of the ComboBox input. */
   setInputValue(value: string): void,
+  /** Function that when called handles updating the ComboBox's input value and selected key. */
   commit(): void
 }
 
 type FilterFn = (textValue: string, inputValue: string) => boolean;
 interface ComboBoxStateProps<T> extends ComboBoxProps<T> {
+  /** The filter function used to determine if a option should be included in the ComboBox list. */
   defaultFilter?: FilterFn,
+  /** Whether the ComboBox allows the menu to be open when the collection is empty. */
   allowsEmptyCollection?: boolean,
+  /** Whether the ComboBox menu should close on blur. */
   shouldCloseOnBlur?: boolean
 }
 
+/**
+ * Provides state management for a ComboBox component. Handles building a collection
+ * of items from props and manages the option selection state of the ComboBox. Also tracks the input value,
```

Comment:
For some reason this `Also` bugs me when I read it. Maybe `In addition...` or `Additional functionality includes...`